koin-projects/gradle.properties

```diff
@@ -28,4 +28,9 @@ org.gradle.caching=true
 # Bintray
 bintray_repo=koin
 bintray_org=ekito
-bintray_github=https://github.com/touchlab/koin
\ No newline at end of file
+bintray_github=https://github.com/touchlab/koin
```

Comment:
wrong one, please change it to `https://github.com/InsertKoinIO/koin`

Reply:
Done!